cedar-drt/fuzz/src/schemas.rs

```diff
@@ -85,14 +107,11 @@ fn action_type_equivalence(name: &str, lhs: ActionType, rhs: ActionType) -> Resu
     }
 }
 
-fn empty_target(spec: &ApplySpec) -> bool {
-    spec.resource_types
-        .as_ref()
-        .map(|v| v.is_empty())
-        .unwrap_or(false)
-        || spec
-            .principal_types
-            .as_ref()
-            .map(|v| v.is_empty())
-            .unwrap_or(false)
+fn is_both_unspecified(spec: &ApplySpec) -> bool {
```

Comment:
Nit: grammatically speaking, I think this should be `are_both_unspecified`. But that looks weird, so I'd suggest changing to just `both_unspecified`.